environments/common.yaml

```diff
@@ -164,7 +164,25 @@ jobs:
       // Filter by name (with regular expression)
       headRegexFilter {
         // Regular expression
-        regex('main|PR-.*')
+        regex('main')
+      }
+    releaseMasterHeadRegexFilter: |-
```

Comment:
Should it be in production.yaml instead, to make it clearer it's a trait different in staging than in production, like the other ones?